lib/Mastodon/System/SessionAuth.vala

```diff
@@ -116,7 +116,7 @@ public class Backend.Mastodon.SessionAuth : Backend.SessionAuth {
     // Create a new session
     string session_id = Uuid.string_random ();
     try {
-      return yield Backend.Session.from_data (session_id, auth_proxy.access_token, auth_server);
+      return yield Backend.Session.from_data (session_id, auth_proxy.access_token, auth_server, true);
```

Comment:
I think `auto_start` should be `false` initially. I would provide a one-window client experience on default, which is also how most users will probably use it, with the setting available for those who want it.

Reply:
That probably makes sense. And stops us auto-opening all of the windows when a user adds multiple accounts. I guess the only question is whether we're consistent on the default window that we open up when there's no auto-start windows.

Also, we'll have to remember to manually begin polling when we start using this for polling/streaming and open a default window because none are auto-start.